tests/licensedcode/data/datadriven/lic3/odc-1.0.text.yml

```diff
@@ -1,2 +1,2 @@
 license_expressions:
-  - odc-by-1.0
+  - ppl
```

Comment:
This is a regression potentially, Maybe [this rule](https://github.com/aboutcode-org/scancode-toolkit/blob/4190-license-licence/src/licensedcode/data/rules/ppl_1.RULE) is not added correctly? @pombredanne ?